setup.py

```diff
@@ -699,48 +376,38 @@ def dependency_order_class_objects(objects: Dict[str, str], custom_types: Dict[s
             if '#' in line:
                 line = line[:line.index('#')]  # strip of comment
             dependencies.extend(re.findall(r'(\w+)', line))  # catch all legible words, potential dependencies
-        dependencies = filter(lambda x: '_' not in x and x.upper() != x, dependencies)  # filter out constants
-        dependencies = filter(lambda x: x not in ignored_dependencies, dependencies)
-        dependencies = filter(lambda x: x not in custom_types, dependencies)
+        dependencies = list(filter(lambda x: '_' not in x and x.upper() != x, dependencies))  # filter out constants
+        dependencies = list(filter(lambda x: x not in ignored_dependencies, dependencies))
+        dependencies = list(filter(lambda x: x not in custom_types, dependencies))
         for dep in dependencies:
             key_list = list(objects.keys())
-            for item in [dep, key] + key_list[key_list.index(dep)+1:]:
+            for item in [dep, key] + key_list[key_list.index(dep) + 1:]:
                 objects[item] = objects.pop(item)
 
 
-def combine_ssz_objects(old_objects: Dict[str, str], new_objects: Dict[str, str], custom_types) -> Dict[str, str]:
-    """
-    Takes in old spec and new spec ssz objects, combines them,
-    and returns the newer versions of the objects in dependency order.
-    """
-    for key, value in new_objects.items():
-        old_objects[key] = value
-    return old_objects
-
-
 def combine_spec_objects(spec0: SpecObject, spec1: SpecObject) -> SpecObject:
     """
     Takes in two spec variants (as tuples of their objects) and combines them using the appropriate combiner function.
     """
     protocols = combine_protocols(spec0.protocols, spec1.protocols)
     functions = combine_dicts(spec0.functions, spec1.functions)
     custom_types = combine_dicts(spec0.custom_types, spec1.custom_types)
-    constant_vars = combine_dicts(spec0.constant_vars, spec1.constant_vars)
-    preset_vars = combine_dicts(spec0.preset_vars, spec1.preset_vars)
+    vars = combine_dicts(spec0.vars, spec1.vars)
     config_vars = combine_dicts(spec0.config_vars, spec1.config_vars)
-    ssz_dep_constants = combine_dicts(spec0.ssz_dep_constants, spec1.ssz_dep_constants)
-    ssz_objects = combine_ssz_objects(spec0.ssz_objects, spec1.ssz_objects, custom_types)
+    ssz_objects = combine_dicts(spec0.ssz_objects, spec1.ssz_objects)
     dataclasses = combine_dicts(spec0.dataclasses, spec1.dataclasses)
+    python_prefix = spec0.python_prefix + spec1.python_prefix
```

Comment:
The linting error, `F811 redefinition of unused 'name' from line n` is self explanatory, so I don't see what benefit a comment here would add. If such a redefinition is needed it can be tagged with `# noqa: F811` .